src/MailChimp/Facade.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Locomotive\MailChimp;
+
+use ReflectionClass;
+use Mailchimp;
+
+/**
+ * File: MailChimp Facade Class
+ *
+ * Acts as a master facade class for MailChimp to allow me to isolate and test
+ * functionality without bootstrapping the vendor API.
+ *
+ * @package Locomotive\MailChimp
+ * @link    https://bitbucket.org/mailchimp/mailchimp-api-php/ Mailchimp API Client
+ * @see     Locomotive\WordPress\Facade {
+ *    Inspired by a modified version of Kevin Sperrine's WordPress facade.
+ * }
+ */
+
+/**
+ * Class: MailChimp Facade
+ *
+ * @version 2015-02-12
+ * @since   0.0.0 (2015-02-12)
+ */
+
+class Facade
+{
+	public  $is_initialized = false;
+	private $facade;
+
+	/**
+	 * MailChimp Initialization
+	 *
+	 * @version 2015-02-12
+	 * @since   2015-02-12
+	 * @access  public
+	 * @param   mixed  The arguments passed to the function
+	 * @return  void|object
+	 */
+
+	public function initialize( $apikey=null, $opts=[] )
```

Comment:
@mcaskill can't we just pass the same arguments needed for Mailchimp here? I get there isn't an equivalent to `call_user_func_array()` for instanciating classes but I'm just not sure it's required in this situation.